### ISSUE TITLE
Improved admin interface: quota used

### ISSUE THREAD

**Issue by Archie1978** [Mailu/Mailu#216]
In the interface, I saw that there was a quota per domain and per user.
But how do we know we have 90% of the quota used in the admin interface?

Of course, I can go into the webmail and see the quota.

But it's always better to see cases before a client calls

**Comment by kaiyou** [comment]
Agreed, this would be a nice addition to the interface. Additionally, total space used per domain could be reported in the domain list.
